setup.cfg

```diff
@@ -1,3 +1,6 @@
 [flake8]
```

Comment:
Oh, I forgot setup.cfg was already in the repo :)

Reply:
Yeah :)